front-end/src/app/player/player.component.css

```diff
@@ -0,0 +1,21 @@
+.player-card {
+    background-color: #e4f7e9;
+    padding: 25px;
+    margin: 15px;
```

Comment:
I have used rem instead of px in the grid component, probably best to stick to one type of unit. Do you prefer px over rem? If so, maybe change the one style in grid.component.css